ShippoTesting/ManifestTest.cs

```diff
@@ -10,8 +10,7 @@ public class ManifestTest : ShippoTest {
         [Test ()]
         public void TestValidCreate ()
         {
-            Manifest testObject = ManifestTest.getDefaultObject ();
-            Assert.AreEqual ("NOTRANSACTIONS", testObject.ObjectStatus);
+			Assert.That (() => ManifestTest.getDefaultObject (),Throws.TypeOf<ShippoException> ());
```

Comment:
Yes, I think we used to return a Manifest with that object status, but now we return a 400